rpms/gn/gn.spec

```diff
@@ -0,0 +1,58 @@
+%global debug_package %{nil}
+
+%global commit 78db9de5aed44f03763176ac6067fb1a73798684
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           gn
+Version:        1559
```

Comment:
I don't see any releases for this project, so I feel you should use the way for [upstream has never choose a version](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_has_never_chosen_a_version). In this case, I suggest you to use 
```
Version:        0
Release:        1.git%{shortcommit}%{?dist}
```

Or similar things like
```
Version:        0
Release:        1.1559git%{shortcommit}%{?dist}
```

Reply:
Adding @cheese to see which is a better version.